### DISCUSSION TITLE
タグミュート機能を作成してほしい

### DISCUSSION THREAD

**Discussion by nagataaaas** [increments/qiita-discussions#158] [要望・アイデア・不具合]
私はQiitaを専ら技術系サイトであるという風に見て使っていたのですが、特にここ2年ほど、ポエム・資格・非技術系のLTのような投稿などなど、関係のないものが多く見受けられます。

そういった記事には共通してつけられるタグがあったりしますので、そのタグをミュートすることで、より快適に技術ライフが遅れるのではないかなと思います。

こういった使い方でなくても、記事的には多いが全く興味のない分野をシャットアウトして、自分の技術領域の記事を多く目にするようにできるというのは、素晴らしいと思います。

**Comment by kskwtnk** [comment]
@nagataaaas 
ご意見ありがとうございます！

タグミュートに関しては多くの声をいただいていて、検討をしています！

私自身欲しいなと思うタイミングはあるのですが、Qiitaの思想として、セットでセレンディピティを上げられるような機能も考えていきたいと思っています
そのためすぐには提供できないかもしれませんが、運営としても入れたい機能であるため進めていきます！

**Comment by nagataaaas** [comment]
ご回答ありがとうございます。

セレンディピティについては素晴らしい理念だと思います。
確かに、自分の意識していなかった(あるいは知らなかったなど)分野についての記事が目に入り、そこから興味を持つということは十分に考えられますし、実際にそういう経験もありました。

しかしそれは、自分の視野外の話題を知ることが出来るという喜びであり、自分が視野に入れたくない話題を知ること では無いと思います。
私個人はですが、資格勉強やマインドセットなどではなく技術を知りたいので、そもそもノイズにしかなっておりません…

タグミュートでなくても、記事と質問に分かれているように記事も「技術記事」と「その他」に分けていただくなど、技術と意識高い系の棲み分けが必要だと感じます。

**Reply by kskwtnk** [reply]
@nagataaaas 
> 自分の視野外の話題を知ることが出来るという喜びであり、自分が視野に入れたくない話題を知ること では無い

こちら、仰る通りだとおもいます 🙏 

タグミュートや記事のジャンル分けなど、それ単体でリリースすることはできるはずなのですが
コミュニティの今後を考えると、できれば「視野から外す」と「新しく視野に入る」をセットにしたいという思いがあります

ただし上記の内容はかなり運営目線であり、ユーザーの皆さまからしたらまた違う考えだろうとも認識しています

極力早く提供できるように議論や実装を進めていくこと自体は揺らぎませんが、考えていることの共有として返信いたします！

**Comment by kyntk** [comment]
@nagataaaas 大変お待たせいたしました。
社内で検討を重ね、先程ベータ版として公開いたしました！
ぜひ使ってみてください。

詳しくはQiita Blogをご確認ください！
https://blog.qiita.com/release-tag-mute-beta/

**Reply by shiracamus** [reply]
タグランキングもミュートしたタグが非表示になるといいですね。

**Reply by kskwtnk** [reply]
@shiracamus 

ご意見ありがとうございます！

現状では、ランキングは「Qiita全体の動向の反映」として、ミュートしたタグも表示するつもりでいます
ただ、非表示にして欲しいという意見もあるだろうとは思っていまして、今後皆さんからの意見も踏まえて表示・非表示を検討していきます！
